src/buffer/out/Row.cpp

```diff
@@ -9,6 +9,8 @@
 #include "textBuffer.hpp"
 #include "../../types/inc/GlyphWidth.hpp"
 
+extern "C" long __isa_enabled;

```

Comment:
Are you stealing somebody else's private symbols leonard

Reply:
_Ah woops I accidentally accessed the CRT's internals. Oh no._

But in all seriousness though, `__isa_available` is an officially supported symbol and part of the `isa_availability.h` system header. I've switched to it for now. I don't quite get why the STL uses `__isa_enabled`.